.github/scripts/patch.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash -x
+
+exit 1 # Remove this lock after updating the following values
```

Comment:
I don't know why I don't see my previous comment here but I will expand. 

I don't know if this is the correct way to force a user to update the version below. Could this be done by passing in an arg into the script and the validating it? 

If the answer is no and we want to leave this as the process then I think we should add a detailed comment here on the steps required before removing the `exit 1`.

Reply:
> I don't know if this is the correct way to force a user to update the version below. Could this be done by passing in an arg into the script and the validating it?

But from where the argument would come from?

Reply:
I will add the comments.